packages/network-of-terms-cli/README.md

```diff
@@ -0,0 +1,55 @@
+# Network of Terms CLI
+
```

Comment:
Perhaps add a short description here, e.g. 'Query the Network of Terms via the command line'?